Mathlib/Analysis/Normed/Lp/ProdLp.lean

```diff
@@ -6,7 +6,7 @@ Authors: Moritz Doll, Sébastien Gouëzel, Jireh Loreaux
 
 import Mathlib.Analysis.MeanInequalities
 import Mathlib.Analysis.Normed.Lp.WithLp
-
+import Mathlib.Tactic.Finiteness
```

Comment:
How many transitive imports are added when `Data/ENNReal/Basic` imports `Finiteness`? Presumably very few, and that would just make it available everywhere.